### PR TITLE
Comment styles

### PR DESCRIPTION
**Pull request by A7DC**

### CODE REVIEW

File: scss/comments.scss [hunk 1]
```diff
@@ -41,130 +20,10 @@ $purple:                #7a43b6 !default;
         margin-bottom: 0;
     }
 
-    .comments-list {
+    ol.comments-list {
```
Comment (tractorcow) [inline scss/comments.scss]: In the default template comments-list is an `<ul>` not an `<ol>`

Comment (A7DC) [inline scss/comments.scss]: This is something I missed, but basically we should never be declaring selectors via their elements (only their classes) to ensure the HTML is able to be modified if it is someway incorrect. I'll remove it now.

File: templates/CommentsInterface_singlecomment.ss [hunk 2]
```diff
@@ -11,32 +11,32 @@
 		<% end_if %>
 	</p>
 <% end_if %>
-
-<div class="comment-text" id="<% if $isPreview %>comment-preview<% else %>$Permalink<% end_if %>">
+	
+<div class="comment-text well" id="<% if $isPreview %>comment-preview<% else %>$Permalink<% end_if %>">
```
Comment (tractorcow) [inline templates/CommentsInterface_singlecomment.ss]: We can drop the `well` from here. It doesn't seem relevant to this template is it?

File: scss/comments.scss [hunk 3]
```diff
@@ -188,3 +73,113 @@ $purple:                #7a43b6 !default;
...
+    white-space: pre-wrap;
+    white-space: pre-line;
+    word-wrap: break-word;
+
+    p:last-child {
+        margin-bottom: 0;
+    }
+}
+
+.date {
```
Comment (tractorcow) [inline scss/comments.scss]: I think this is too generic a style to have at the top level. There could be any `date` in the html not related to comments. Can we nest these similar to the previous styles, in `.comments-holder-container`?

File: scss/comments.scss [hunk 4]
```diff
@@ -188,3 +73,113 @@ $purple:                #7a43b6 !default;
...
+    }
+}
+
+&.unmoderated {
+    > .comment-text {
+        border: 1px solid yellow;
+    }
+}
+
+.info {
```
Comment (tractorcow) [inline scss/comments.scss]: Same as comment on .date

File: scss/comments.scss (hunk 3 above)
Comment (A7DC) [inline scss/comments.scss]: The .date is already nested within a .comment class. Previously it was nested both in a .comment class and a comment-container class. There is no need for it to be that specific.

File: scss/comments.scss (hunk 4 above)
Comment (A7DC) [inline scss/comments.scss]: Same comment as above.

File: scss/comments.scss (hunk 3 above)
Comment (tractorcow) [inline scss/comments.scss]: Ok, well it seems like there's an indentation issue. Can you please fix that?

File: scss/comments.scss [hunk 5]
```diff
@@ -188,3 +73,113 @@ $purple:                #7a43b6 !default;
...
+.comment {
+    clear:both;
+    list-style-type: none;
+    overflow: hidden;
+    padding: 20px 0 10px;
+    position: relative;
+
+    &.author-comment {
+        &:after {
+          content: 'Author';
```
Comment (tractorcow) [inline scss/comments.scss]: It looks like the indentation issues start about here.

File: scss/comments.scss [hunk 6]
```diff
@@ -188,3 +73,113 @@ $purple:                #7a43b6 !default;
...
+          float: right;
+          position: absolute;
+          top: 1.5em;
+          right: 0.5em;
+          font-size: 1em;
+          font-weight: bold;
+      }
+
+      > .comment-text {
```
Comment (tractorcow) [inline scss/comments.scss]: Almost there :D Still looks a bit out here.

If you set your indentation settings to 4/4 and use-tabs maybe it'll simplify the indentation?

**Comment by tractorcow** [comment]
Squashed and merged with https://github.com/silverstripe/silverstripe-comments/commit/651a4e4977055bdf00a43d6db752dde730097766
